documentation/whatsnew/fusetools/fusetools-news-11.0.0.AM2.adoc

```diff
@@ -0,0 +1,48 @@
+= Fuse Tooling - What's New in 11.0
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 11.0.0.AM2
+:page-product_id: jbt_core
+:page-product_version: 4.6.0.AM2
+
+== Camel Rest DSL from WSDL wizard
+
+There is a new _"Camel Rest DSL from WSDL"_ wizard. This wizard wraps the wsdl2rest tool now included with the Fuse 7 distribution, which 
```

Comment:
might worth adding a link on wsdl2rest goign to https://github.com/jboss-fuse/wsdl2rest so that users which doesn't know about it can have an eye